pkg/transferer/import_test.go

```diff
@@ -245,7 +263,7 @@ func writeFakeFile(t *testing.T, w io.Writer, opts fakeFileOpts) {
 			Typeflag: tar.TypeReg,
 			Name:     path.Join("ch", fmt.Sprintf("chunk-%d.bin", i)),
 			Size:     chunkSize,
-			Mode:     0600,
+			Mode:     0o600,
```

Comment:
I remember that I had an issue with changing mode to this type with setting incorrect mode, please make sure that it works as expected.

Reply:
We are using this format all over PMM, it works fine AFIK.